paletteView.js

```diff
@@ -0,0 +1,106 @@
+class PaletteView {
+  constructor(
+    input,
+    form,
+    container,
+    lockIcon,
+    copyIcon,
+    arrowIcon,
+    lockState
+  ) {
+    this.input = input;
+    this.form = form;
+    this.container = container;
+    this.lockIcon = lockIcon;
+    this.copyIcon = copyIcon;
+    this.arrowIcon = arrowIcon;
+    this.lockState = lockState;
+    this.setupFormAction();
+    this.copyToClipBoard();
+    this.arrowClick();
+    this.lockClick();
+  }
+
+  setupFormAction() {
+    this.form.addEventListener("submit", (ev) => {
```

Comment:
Why can I use anonymous event listeners here, but in PaletteController I had to bind keypress with a new View's body? Could I have not used a View class? Should I have bound this to the View's keypress?